### ISSUE TITLE
security management update

### ISSUE THREAD

**Issue by JeanBarriere** [storyscript/dashboard#145]
**Based from documentation and security review;**

tl;dr -- we need to update how cookies are treated across the platform.

### Observations
As a user,
I need to hit /user endpoint to get my secrets, **if** I'm logged in.
I should be able to know I'm having a valid session without hitting a network request.

### Acceptance
**Given** a user access to the dashboard,
**When** we load the app,
**Then** it checks a `expiration-cookie` to know if a user is logged in, before hitting any request.

### Notes
Apart from adding this cookie, we **have to update** the auth cookie to add a SameSite:strict to it, so we are CSRF-exploit free, and broadcast the logout through tabs (which isn't the case today)

_Originally posted by @JeanBarriere in https://github.com/storyscript/dashboard/issues/122#issuecomment-535026081_
